src/github.com/getlantern/byteexec/byteexec.go

```diff
@@ -32,13 +31,11 @@ import (
 	"github.com/getlantern/golog"
 )
 
-const (
-	fileMode = 0744
-)
-
 var (
 	log = golog.LoggerFor("Exec")
 
+	fileMode = os.FileMode(0744)
+
```

Comment:
I changed this to an actual FileMode type for better error logging in the unit test.